TestProject/ClientApp/src/components/PostPage.js

```diff
@@ -2,19 +2,10 @@
 import { NotificationContainer, NotificationManager } from 'react-notifications';
 import 'react-notifications/lib/notifications.css';
 import Rating from "./Rating";
+import getRating from '../services/RatingService';
 
-function PostPage({ postId, userProfile, getUpdatedRating }) {
-    const [post, setPost] = useState({
-        id: 0,
-        title: "",
-        content: "",
-        createDate: "",
-        updateDate: "",
-        updateRating: "",
-        author: { firstName: "", lastName: "" },
-        selectedRating: 0,
-        totalRating: 0
-    });
+function PostPage({ postId, userProfile }) {
+    const [post, setPost] = useState(undefined);
```

Comment:
You never need to set `undefined`, it is default value. It's better use an empty object in this case.